rpcs3/Emu/System.cpp

```diff
@@ -1403,6 +1416,15 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 		else if (m_cat == "DG" && from_hdd0_game && disc.empty())
 		{
 			// Disc game located in dev_hdd0/game
+			bdvd_dir = g_cfg_vfs.get(g_cfg_vfs.dev_bdvd, rpcs3::utils::get_emu_dir());
+
+			if (!fs::is_dir(bdvd_dir) || !std::filesystem::is_empty(bdvd_dir))
```

Comment:
if it's always an empty directory, why be able to configure it's location?

Reply:
Both scenarios should be very rare.
One is mainly for devs and useless in my opinion, but i kept it in to not enrage someone.
The other only is needed if you play a handful of games.